### PR TITLE
bug fix: removed border from solution code

### LINKED ISSUE #50890: Remove border from solution code
### Describe the Issue

Unnecessary borders are displayed around each line in the solution code of certification projects.

### Affected Page

For example: https://www.freecodecamp.org/certification/sidemt/javascript-algorithms-and-data-structures

### Steps to Reproduce

1. Go to someone's certification page
2. Click on 'View' (> 'View Code') to see the solution code

### Expected behavior

Each line in the solution code should not have borders around it

### Screenshots

image

### System

- Device: Laptop
- OS: Windows 10
- Browser: Chrome
- Version: 114.0.5735.199 (Official Build) (64-bit)

### Additional context

The border seems to be added in #50693

### PR DESCRIPTION
**Pull request by blu3IceCube**
**Description**

I removed the border property from the code tag in the global.css file which removed the border from the selection code.

**Related Issues**

Closes #50890

**Checklist**

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.
- [x] All new and existing tests pass locally and on GitPod.

### CODE REVIEW

**Comment by Sboonny** [comment]
Changing the bootstrap file isn't the right approach to solving this issue. 

To solve the issue move the classes added in https://github.com/freeCodeCamp/freeCodeCamp/pull/50693/files, to CSS selector that affects the challenges' layout, or not affect the setting layout.

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -460,6 +461,7 @@ input:focus {
 :not(pre) > code {
     border: 1px solid var(--secondary-color);
```
Comment (naomi-lgbt) [inline client/src/components/layouts/global.css:462]:
```suggestion
  border: 1px solid var(--secondary-color);
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Should please the linter.

**Comment by naomi-lgbt** [comment]
Thank you for your interest in contributing.

We typically merge the first pull request that comprehensively resolves the issue. As such, we are closing this in favour of #50968.

**Pull request closed without merging**